buildenv/jenkins/Jenkinsfile_personal

```diff
@@ -2,8 +2,8 @@
 pipeline {
     agent {label "${getPlatformAndLabel(SPEC)[1]}"}
 	parameters {
-		string (defaultValue: "AdoptOpenJDK", description: 'personal repo from AdoptOpenJDK/openjdk-tests', name: 'personalRepo')
-		string (defaultValue: "master", description: 'personal branch', name: 'personalBranch')
+		string (defaultValue: "https://github.com/AdoptOpenJDK/openjdk-tests.git", description: 'personal repo from AdoptOpenJDK/openjdk-tests', name: 'ADOPTOPENJDK_REPO')
+		string (defaultValue: "master", description: 'personal AdoptOpenJDK branch', name: 'ADOPTOPENJDK_BRANCH')
 		choice (choices: 'linux_x86-64\nlinux_x86-64_cmprssptrs\nmac_x86-64\nlinux_390-64\nlinux_390-64_cmprssptrs\nlinux_ppc-64_le\nlinux_ppc-64_cmprssptrs_le\nlinux_arm\nwin_x86-64\nwin_x86-64_cmprssptrs\naix_ppc-64\naix_ppc-64_cmprssptrs', description: 'SPEC?', name: 'SPEC')
 		choice (choices: 'SE80\nSE90\nSE100', description: 'What is JAVA_VERSION?', name: 'JAVA_VERSION')
```

Comment:
we may eventually want to add the other values in the choice list... amber, valhalla, panama